test/Maqduni.AspNetCore.Identity.RavenDb.Tests/TestStores.cs

```diff
@@ -111,6 +111,49 @@ public void RoleCreateAsync(string roleName)
             Assert.True(result.Succeeded);
         }
 
+        [Theory(DisplayName = "Role AddClaimsAsync")]
+        [InlineData("Admin", "GraduatedSchoolYear", "2006")]
+        [InlineData("Admin", "GraduatedUniversityYear", "2010")]
+        public void RoleAddClaimsAsync(string roleName, string claimType, string claimValue)
+        {
+            var role = _roleStore.FindByNameAsync(roleName).Result;
+            Assert.NotNull(role);
+
+            var claim = new Claim(claimType, claimValue);
+            _roleStore.AddClaimAsync(role, claim).Wait();
+
+            var claims = _roleStore.GetClaimsAsync(role).Result;
+            Assert.True(claims.Any(c => c.Type.Equals(claim.Type, StringComparison.OrdinalIgnoreCase) && c.Value.Equals(claim.Value, StringComparison.OrdinalIgnoreCase)));
+
+            var result = _roleStore.UpdateAsync(role).Result;
+            Assert.True(result.Succeeded);
+        }
+
+        [Theory(DisplayName = "Role WithClaimsRemoveClaimAsync")]
+        [InlineData("Admin", "GraduatedSchoolYear", "2006")]
+        [InlineData("Admin", "GraduatedUniversityYear", "2010")]
+        public void RoleWithClaimsRemoveClaimAsync(string roleName, string claimType, string claimValue)
+        {
+            var role = _roleStore.FindByNameAsync(roleName).Result;
+            Assert.NotNull(role);
+
+            for (int i = 0; i < 10; i++)
```

Comment:
Though it's a good way to point out that claims can be duplicated and by deleting one, all of the dupes will also be gone.

Disregard my first comment, don't make any changes here.